packages/web-components/src/components/ic-side-navigation/ic-side-navigation.css

```diff
@@ -261,10 +261,10 @@ slot[name="app-icon"]::slotted(a) {
 
 .navigation-landmark-title {
   position: absolute;
-  width: 0.063rem;
-  height: 0.063rem;
+  width: var(--ic-space-1px);
+  height: var(--ic-space-1px);
   padding: 0;
-  margin: -0.063rem;
+  margin: -var(--ic-space-1px);
```

Comment:
I don't think the negative number will work like this. It needs to be multiplied by `-1`: 
`margin: calc(-1 * var(--ic-space-1px))`